python/pyspark/sql/column.py

```diff
@@ -151,6 +162,8 @@ def __init__(self, jc):
     __rdiv__ = _reverse_op("divide")
     __rtruediv__ = _reverse_op("divide")
     __rmod__ = _reverse_op("mod")
+    __pow__ = _bin_func_op("pow")
```

Comment:
We already have `pow` in pyspark.sql.functions.

For here, it's easy to do like this:
```
from pyspark.sql.function import pow
__pow__ = pow
__rpow__ = lambda c, other: pow(other, c)
```